doc/QuickStart.md

```diff
@@ -29,23 +32,48 @@ Installing
 
 ### Prerequisites
 
-#### Installing CPAN modules
+#### Requirements *(root)*
+
+	$ dnf -y install perl perl-CPAN git screen crontabs nano
+> or 		$ aptitude install perl perl-CPAN git screen crontabs nano
+> or 		$ apt-get install perl perl-CPAN git screen crontabs nano
+
+	$ adduser -m pbot
```

Comment:
Local operating system is beyond the scope of this QuickStart guide. Users are expected to understand where they want to install PBot. It is up to them whether to create a new user or to use /usr/local/bin or to use their current user home directory or etc.